tests/test_delayed.py

```diff
@@ -0,0 +1,103 @@
+import time
+from pytest_testconfig import config as testconfig
+from kubernetes import client
+
+from tests.deployment import create_deployment, delete_deployment
+from tests.fixtures import set_namespace, load_config, init_session, set_docker_images, session_id, DeploymentInfo, init_session
+from tests.test_bs import setup_clients, save_log_on_exit, setup_bootstrap, create_configmap
+from tests.test_bs import current_index, wait_genesis, GENESIS_TIME, BOOT_DEPLOYMENT_FILE, CLIENT_DEPLOYMENT_FILE, get_conf
+from tests.misc import CoreV1ApiClient
+from tests.queries import ES, query_message
+from elasticsearch_dsl import Search, Q
+
+# ==============================================================================
+#    TESTS
+# ==============================================================================
+
+def setup_clients_in_namespace(namespace, bs_deployment_info, client_deployment_info, client_config,
```

Comment:
@gavraz Why have you duplicated this code from test_bs.py?

Reply:
Please create the fixtures/functions central python file and I'll use it (in this PR or after it is ready) as we talked.